lib/libvarnish/vev.c

```diff
@@ -59,6 +59,7 @@ struct vevsig {
 	struct vev		*vev;
 	struct sigaction	sigact;
 	unsigned char		happened;
+	siginfo_t		siginfo[1];
```

Comment:
I think it's like that because with @dridi we wanted not to include signal.h in vev.h by using an opaque pointer, since we could not do that this approach might not be the good one indeed, I have added a commit which moves siginfo from  vevsig to vev, if it's fine I'll use it to edit the first commit, otherwise we'll trash it

Reply:
I think this is an improvement, thank you.